src/Service/ServiceUtils.cpp

```diff
@@ -268,9 +273,27 @@ static ErrorMessageOr<fs::path> FindSymbolsFilePathInStructuredDebugStore(
       absl::StrFormat("File does not exist: %s", path_in_structured_debug_store.string())};
 }
 
-ErrorMessageOr<fs::path> FindSymbolsFilePath(
-    const fs::path& module_path, const std::vector<fs::path>& search_directories) noexcept {
-  OUTCOME_TRY(module_elf_file, CreateElfFile(module_path.string()));
+static ErrorMessageOr<fs::path> FindSymbolsFilePathCoff(const fs::path& module_path,
+                                                 CoffFile* module_coff_file) noexcept {
+  if (module_coff_file == nullptr) {
+    return ErrorMessage("module_coff_file is nullptr");
```

Comment:
should this be a check?

Reply:
Yeah, it probably should. Will get triggered if the dynamic cast failed (at least, the way it is used right now).